### PR TITLE
thrall check

### PR DESCRIPTION
**Pull request by Aurorablade**
fixes https://github.com/ParadiseSS13/Paradise/issues/7279

:cl: Fethas
bugfix: Makes thrall examine use MASKCOVERMOUTH and HIDEFACE flags
/ :cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -443,7 +443,7 @@
 		msg += "<span class='warning'><b>[src] has \a [implant] sticking out of [t_his] flesh!</b></span>\n"
 	if(digitalcamo)
 		msg += "[t_He] [t_is] repulsively uncanny!\n"
-	if(!wear_mask && is_thrall(src) && in_range(user,src))
+	if(!(wear_mask.flags_inv & HIDEFACE || wear_mask.flags_cover & MASKCOVERSMOUTH) && is_thrall(src) && in_range(user,src))
```
Comment (FalseIncarnate) [inline code/modules/mob/living/carbon/human/examine.dm]: Might want to check for `skipface` here, so you don't magically see through their hardsuit helmet.

**Comment by Fox-McCloud** [comment]
@Crazylemon64 

Not really sure if this is a balance change or a bugfix. I'll leave it at your discretion.

**Comment by deep-fried-faceplant** [comment]
I think it's primarily an oversight, with the balance change being a side effect. Sometimes, fixing bugs affects balance. I doubt it will severely nerf thralls though, considering how readily available face hiding masks are. Things like cigarettes and bandanas around the neck shouldn't block the face.

**Comment by scrubmcnoob** [comment]
unless your smoking some real beefy cigarettes, i don't think it was intended for those things to hide your facial features

**Comment by taukausanake** [comment]
Smoke so hard your face is hidden behind the fumes

**Comment by Crazylemon64** [comment]
vape life

**Pull request merged**